dist/21-526EZ-schema.json

```diff
@@ -2274,15 +2330,28 @@
     },
     "serviceInformation": {
       "type": "object",
+      "required": [
+        "servicePeriods",
+        "servedInCombatZone"
```

Comment:
Does this have a UI implication as well?

Reply:
I think it does...I'm tracking UI changes as a result of this Yet Another Schema Update in a separate issue.